src/kontrol/utils.py

```diff
@@ -112,3 +104,40 @@ def kontrol_file_contents() -> str:
 $ kontrol command --help
 ```
 """
+
+
+def kontrol_toml_file_contents() -> str:
+    return """[build.default]
+foundry-project-root       = '.'
+regen                      = true
+rekompile                  = true
+verbose                    = true
+debug                      = false
+require                    = 'lemmas.k'
+module-import              = 'TestBase:KONTROL-LEMMAS'
+
+[prove.default]
+foundry-project-root       = '.'
+verbose                    = true
+debug                      = false
+max-depth                  = 25000
+reinit                     = false
+cse                        = false
+workers                    = 4
+failure-information        = false
```

Comment:
I think we might want `failure-information` to be true by default, as it prints the failing node with the path condition? Also, IIRC, counterexample might not get printed if `failure-information` is off even with `counterexample-information` being true (but not 100% sure).